test/fixtures/track-purchase2.json

```diff
@@ -24,13 +24,13 @@
       "currency": "USD",
       "discount": 0,
       "email": "test@iterable.com",
-      "orderId": "R550311537",
+      "order_id": "R550311537",
```

Comment:
got it. 

it's not a reserved field for us, so it won't affect interaction with our api.